### PR TITLE
[kommander] chore: allow adding labels to kommander namespace

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

chore

**What this PR does/ why we need it**:

Allow appending namespaces to the deployed namespace where kommander lives

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
tested this locally

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/kommander/templates/namespace-labeler.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    spec:
+      serviceAccountName: namespace-label
+      containers:
+        - name: namespace-label
+          image: "bitnami/kubectl:1.19.7"
```
Comment (s12chung) [inline stable/kommander/templates/namespace-labeler.yaml:18]: forgot if this image had a CVE

**Review by s12chung — APPROVED** [review]
just a note. otherwise, lgtm.

File: stable/kommander/templates/namespace-labeler.yaml (hunk 1 above)
Comment (gracedo) [reply stable/kommander/templates/namespace-labeler.yaml:18]: this image should be free of cves

**Review by gracedo — APPROVED** [review]

File: stable/kommander/templates/namespace-labeler.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          command: ["kubectl"]
+          args: ["label", "namespace", "{{ .Release.Namespace }}", {{ join "," .Values.namespaceLabels  }} ]
```
Comment (joejulian) [inline stable/kommander/templates/namespace-labeler.yaml:20]: This job will fail if the namespace does not exist. That _is_ what we want, though, isn't it?

Comment (alejandroEsc) [reply stable/kommander/templates/namespace-labeler.yaml:20]: Yes it is, in addition if we do this post-install we would defeat the purpose of adding the label to the namespace. Do you suspect that the .Release.Namespace may be empty? such as in the default case?

**Pull request merged**